_includes/releases/v22.1/v22.1.19.md

```diff
@@ -0,0 +1,62 @@
+
+<h3 id="v22-1-19-db-console-changes">DB Console changes</h3>
+
+- New data is now auto-fetched every 5 mins on the [**Statement and Transaction Fingerprints**](../v22.1/ui-statements-page.html#statement-fingerprint-page) pages. [#100702][#100702]
+
+<h3 id="v22-1-19-bug-fixes">Bug fixes</h3>
+
+- Previously, [`ADD COLUMN ... DEFAULT cluster_logical_timestamp()`](../v22.1/alter-table.html) would crash the node and leave the table in a corrupt state. The root cause is a `nil` pointer dereference. The bug is now fixed by returning an unimplemented error and hence disallowing using the [builtin function](../v22.1/functions-and-operators.html#array-functions) as the default value when backfilling. [#99682][#99682]
+- Fixed a bug where the stats columns on the [**Transaction Fingerprint Overview**](../v22.1/ui-transactions-page.html#transaction-details-page) page was continuously incrementing. [#99405][#99405]
+- Fixed a bug that prevented the garbage-collection job for the [`TRUNCATE TABLE`](../v22.1/truncate.html) command from successfully finishing if the table descriptor had already been garbage-collected. The garbage-collection job now succeeds in this situation by handling the missing descriptor edge case. [#100146][#100146]
+- Fixed a bug present in v21.1 that would cause the SQL gateway node to crash if a [view was created](../v22.1/create-view.html) with circular or self-referencing dependencies. [#100165][#100165]
+- Fixed a bug in evaluation of `ANY`, `SOME`, and `ALL` [sub-operators](../v22.1/functions-and-operators.html#operators) that would cause expressions like `NULL = ANY(ARRAY[]::INT[])` to return `NULL` instead of `FALSE`. [#100363][#100363]
+- Fixed a bug that could prevent a cached query with a [user-defined type](../v22.1/create-type.html) reference from being invalidated even after a schema change that should prevent the type from being resolved. [#100358][#100358]
+- Fixed a bug existing before v22.1 that could cause a projected expression to replace column references with incorrect values. [#100368][#100368]
+- Fixed a bug where the physical disk space of some tables could not be calculated. [#100937][#100937]
+- Fixed a bug such that the `crdb_internal.deserialize_session` function works properly with prepared statements that have more param type hints than params. [#101363][#101363]
+- Fixed a bug where in PostgresSQL extended protocol mode it was possible for auto-commits to not execute certain logic for DDL, when certain DML ([`INSERT`](../v22.1/insert.html)/[`UPDATE`](../v22.1/update.html)/[`DELETE`](../v22.1/delete.html)) and DDL were combined in an implicit transaction. [#101630][#101630]
+- In the [**DB Console Stats**](../v22.1/ui-statements-page.html) pages, issuing a new request for stats while one is pending is now allowed and will replace the pending request. [#100702][#100702]
```

Comment:
I would maybe refer to SQL Activity from the left-hand navigation and then the tabs. It might be worth checking out the Looms in the PR, which show the relevant pages/tabs here. "DB Console Stats" was hard to search for/find anything relevant in the link.